hw_3.js

```diff
@@ -0,0 +1,133 @@
+"use strict"
+
+// Task 1
+
+let message = 'true';
+let booling = false;
+let age = 17;
+let user = undefined;
+let num = null;
+
+console.log(typeof message);
+console.log(typeof booling);
+console.log(typeof age);
+console.log(typeof user);
+console.log(typeof num);
+
+// Task 2
+
+let height = 15;
+let width = 20;
+
+let maxNum = [height , width];
+
+let m = Math.max.apply(null , maxNum);
+
+console.log(m);
```

Comment:
так функции через apply практически никогда не используют
тут хватило бы
```suggestion
let height = 15;
let width = 20;

let m = Math.max(height, width);

console.log(m);
```

Reply:
это на будущее,  а так все норм